reactor-test/src/main/java/reactor/test/util/RaceTestUtils.java

```diff
@@ -51,9 +51,9 @@
 			Predicate<? super T> stopRace,
 			BiPredicate<? super T, ? super T> terminate) {
 
-		Scheduler.Worker w1 = Schedulers.elastic()
+		Scheduler.Worker w1 = Schedulers.boundedElastic()
```

Comment:
since for the race testing we really do need two legit threads, I'd consider using `Schedulers.newSingle()` _or_ a `newBoundedElastic(2...)` in order to ensure no overlap with the default one.

Reply:
Not sure I follow: wasn't the problem already there when using `elastic()`? 

Also, to be 100% clear, you recommend changing both lines to `Schedulers.newSingle().createWorker()`, correct?

Reply:
It wasn't, because the `elastic()` scheduler would blindly create 1 `Thread` per `Worker`, whereas the `boundedElastic()` will ultimately limit the number of created threads. yes, `Schedulers.newSingle().createWorker()`. Arguably the code could use a simple `fixedSizedThreadPoolExecutor` too.

Reply:
>It wasn't, because the elastic() scheduler would blindly create 1 Thread per Worker, whereas the boundedElastic() will ultimately limit the number of created threads. 

Gotcha!